src/mumble/AudioOutputToken.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_AUDIOOUTPUTTOKEN_H_
+#define MUMBLE_MUMBLE_AUDIOOUTPUTTOKEN_H_
+
+#include <AudioOutput.h>
+#include <AudioOutputToken.h>
+
+class AudioOutputBuffer;
+
+class AudioOutputToken {
+private:
+	AudioOutputBuffer *m_buffer;
+
+	friend class AudioOutput;
+
+public:
+	AudioOutputToken(AudioOutputBuffer *buffer) : m_buffer(buffer){};
+	~AudioOutputToken(){};
+
+	inline bool operator==(const AudioOutputToken &rhs) const { return m_buffer == rhs.m_buffer; }
+	inline bool operator!=(const AudioOutputToken &rhs) const { return m_buffer != rhs.m_buffer; }
+
+	static AudioOutputToken None() { return AudioOutputToken(nullptr); }
```

Comment:
```suggestion
	operator bool() const { return m_buffer; }
```
(I think that was the syntax for user-defined conversions)